### ISSUE TITLE
Warnings for Pdag.get_datasets()

### ISSUE THREAD

**Issue by michaelaye** [msbentley/psa_utils#2]
When executing the `get_datasets()` call I get these warnings:

```
dsets = p.get_datasets()
WARNING: W42: None:1:81: W42: No XML namespace specified [astropy.io.votable.tree]
WARNING: W15: None:2:46: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:161: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:282: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:427: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:554: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:681: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:829: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:956: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:1068: W15: FIELD element missing required 'name' attribute [astropy.io.votable.tree]
WARNING: W15: None:2:1177: W15: FIELD element missing required 'name' attribute (suppressing further warnings of this type...) [astropy.io.votable.tree]
```

**Comment by msbentley** [comment]
Thanks @michaelaye! It looks like these are valid warnings. If I get the VoTable:

```
curl -o votable.xml https://archives.esac.esa.int/psa/pdap/metadata?RETURN_TYPE=VOTABLE&RESOURCE_CLASS=DATA_SET
```
and run volint on it, I get:

```
$ volint votable.xml 
Validation report for votable.xml

1: W42: No XML namespace specified
...tp://us-vo.org/xml/VOTable.dtd"><VOTABLE version="1.1"><RESOURCE...
                                   ^

2: W15: FIELD element missing required 'name' attribute
<INFO name="QUERY_STATUS" value="OK" /><TABLE><FIELD ID="DATA_SET.D...
                                              ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.DATA_SET_NAM...
                                   ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.DATA_ACCESS_...
                                   ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.XML_DESCRIPT...
                                   ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.PRODUCER.FUL...
                                   ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.PRODUCER.INS...
                                   ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.PRODUCER.NOD...
                                   ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.START_TIME" ...
                                   ^

2: W15: FIELD element missing required 'name' attribute
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.STOP_TIME" u...
                                   ^

2: W15: FIELD element missing required 'name' attribute (suppressing
  further warnings of this type...)
... datatype="char" arraysize="*"/><FIELD ID="DATA_SET.NPRODUCTS" u...
                                   ^
```
I'll have to dig into the PDAP spec to see if this is related to a change in the VOTable format over the years, or something else. Otherwise we can supress these warnings.

**Comment by msbentley** [comment]
Closed by 0b07abbbec38184783b653e3725669584b31de66
